plugin/system_copy.vim

```diff
@@ -22,6 +26,12 @@ function! s:system_copy(type, ...) abort
   echohl String | echon 'Copied to system clipboard via: ' . mode | echohl None
 endfunction
 
+function! s:system_paste() abort
+  let command = <SID>PasteCommandForCurrentOS()
+  put =system(command)
+  echohl String | echon 'Pasted to vim using: ' . command | echohl None
```

Comment:
I think we can remove this line. For me at least, Vim's own messages are displayed, not this line.